index.js

```diff
@@ -27,14 +27,26 @@ const outfield = (lineup) => {
   return lineup.position === 'OF'
```

Comment:
The way you did this entire project is quite impressive.  Only thing is that your position checker only looks for if there are the correct number of outfielders what happens if there are 2 catchers?  The test passed since the only thing that was altered was the outfield but the function should look for the appropriate amount of each position.  Only taking 2 points and again the rest is impressive.